src/azure-cli/azure/cli/command_modules/network/_params.py

```diff
@@ -812,10 +812,10 @@ def load_arguments(self, _):
     with self.argument_context('network private-endpoint') as c:
         c.argument('private_endpoint_name', private_endpoint_name, options_list=['--name', '-n'])
         c.argument('location', get_location_type(self.cli_ctx), validator=get_default_location_from_resource_group)
-        subnet_help = get_folded_parameter_help_string('subnet', other_required_option='--vnet-name')
+        subnet_help = get_folded_parameter_help_string('subnet', other_required_option='--vnet-name') + ' and do not specify the --vnet-name'
```

Comment:
Can we also validate this logic in validator as well?

Reply:
we indeed tell the exclusion in 'help message', but not in the same message block. Maybe the issue owner think it should be more obvious.